Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y fio jq wget \
```

Comment:
It is also simpler to install this via apk, no cleanup is necessary :) 
`apk add --no-cache fio==3.13-r1`